### PR TITLE
feat: adding config helper

### PR DESCRIPTION
**Pull request by bblxssm**

### CODE REVIEW

**Comment by bblxssm** [comment]
This makes users not complain by having an invalid config 😶

File: confighelper.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                display: block;
+                width: 20%;
+                padding: 20px;
+            }           
+        </style>
+    </head>
+
+    <body>
+        <h1>CapsuleFarmerEvolved config helper</h1>
+        <h3>Use this to generate your config file. <br>The account name will only be visible in the software.<br>The username and password are the ones you use to log into <a href="https://lolesports.com">lolesports.com</a><br>Your login information, or any other data are not sent to us, the internet, or any third party outside of your personal computer.<br>(Open sourced on github: <a href="https://github.com/bblxssm/confighelper">click here</a>)<br>When you're done with adding accounts, copy the config below and paste it into config.yaml<br>Alternatively, you can use the Download button under the config to download it.<br>For any issues open a ticket in the help channel.<br>- Six#9552</h3>
```
Comment (DjordyKoert) [inline confighelper.html:17]: This should probably not link to your own account

File: confighelper.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            function download() {
+            var hiddenElement = document.createElement('a');
+
+            hiddenElement.href = 'data:attachment/text,' + encodeURI(toDownload);
+            hiddenElement.target = '_blank';
+            hiddenElement.download = 'config.yaml';
+            hiddenElement.click();
+        }
+
+        document.getElementById('test').addEventListener('click', download);
```
Comment (DjordyKoert) [inline confighelper.html:51]: Fix indent

File: confighelper.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <label for="password">Riot password:</label>
+        <input type="password" name="password" id="password">
+        <button onclick="addAccount()">Add account</button>
+
+        <h2>Your config:</h2>
+        <code id="config">accounts:</code>
+        <label for="test">Or alternatively: </label>
+        <button name="test" id="test">Download</button>
+        <script>
+            var toDownload = "accounts:"
```
Comment (DjordyKoert) [inline confighelper.html:31]: I recommend using either `let` or `const` instead of `var`

**Comment by DjordyKoert** [comment]
@LeagueOfPoro I recommend fixes these small changes (maybe revert?)

File: confighelper.html (hunk 1 above)
Comment (bblxssm) [reply confighelper.html:17]: It was planned to be uploaded to my github so thats why it was linked there. I will submit another request and correct things.

**Pull request merged**